client/src/reducers/continuousSelection.js

```diff
@@ -0,0 +1,25 @@
+import { makeContinuousDimensionName } from "../util/nameCreators";
+
+const ContinuousSelection = (state = {}, action) => {
+  switch (action.type) {
+    case "reset World to eq Universe": {
+      return {};
+    }
+    case "continuous metadata histogram start":
+    case "continuous metadata histogram brush":
+    case "continuous metadata histogram end": {
+      const name = makeContinuousDimensionName(
+        action.continuousNamespace,
+        action.selection
+      );
+      const continuousSelection = { ...state };
+      continuousSelection[name] = action.range;
```

Comment:
You could use computed property names `computed key` to keep the state return statement in the usual fashion:
```
const continuousSelection = { ...state, [name]: action.range };
```

Reply:
will fix